### PR TITLE
Docs: Heading improvements

### PR DESCRIPTION
**Pull request by ponori**
A few small tweaks to the Heading docs to clean up visuals and add some clarifications.

### CODE REVIEW

File: docs/pages/heading.js [hunk 1]
```diff
@@ -377,7 +379,10 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
 `}
           />
         </MainSection.Subsection>
-        <MainSection.Subsection title="Alignment">
+        <MainSection.Subsection
+          title="Alignment"
+          description="Use `align` to adjust the positioning of text within wrapper elements."
```
Comment (dangerismycat) [inline docs/pages/heading.js:384]: s/"wrapper"/"container"

File: docs/pages/heading.js [hunk 2]
```diff
@@ -399,9 +404,10 @@ export default function DocsPage({ generatedDocGen }: {| generatedDocGen: DocGen
         </MainSection.Subsection>
         <MainSection.Subsection
           title="Accessibility level"
-          description="For accessibility purposes, we allow you to override the heading level.
+          description={`
+For accessibility purposes, we allow you to override the heading level.
 
-        We should have one level 1 per page &amp; levels should be appropriately nested. E.g. level 1 followed by level 2 &amp; level 2 followed by level 2 or level 3. We also allow headings without an accessibility level."
+We should have one level 1 per page &amp; levels should be appropriately nested. E.g. level 1 followed by level 2 &amp; level 2 followed by level 2 or level 3. We also allow headings without an accessibility level. We recommend against using \`accessibilityLevel="none"\`.`}
```
Comment (dangerismycat) [inline docs/pages/heading.js:410]: These two sentences read a bit awkward together. Consider combining to something like "For certain specific situations, it is possible to use Heading with an accessibility level; however, we recommend against using this if possible."

**Review by dangerismycat — APPROVED** [review]

File: docs/pages/heading.js (hunk 2 above)
Comment (ponori) [reply docs/pages/heading.js:410]: Done!

File: docs/pages/heading.js (hunk 1 above)
Comment (ponori) [reply docs/pages/heading.js]: Not sure I follow what this means.

Comment (dangerismycat) [reply docs/pages/heading.js]: Sorry, [`sed` syntax](https://en.wikipedia.org/wiki/Sed#Substitution_command): I'm suggesting that we replace "wrapper" with "container", which better describes the parent element.

Comment (ponori) [reply docs/pages/heading.js]: Oh, nice! TIL! Thanks and fixed!

**Pull request merged**